### DISCUSSION TITLE
Uninstalling 'VS Code Server' from Proxmox host

### DISCUSSION THREAD

**Discussion by dimatx** [tteck/Proxmox#410] [Help Needed] [answered]
I didn't realize VS Code Server script is meant to be run inside of an LXC, and ran it in host, thinking it'd create an LXC as part of the process. How would I go about uninstalling VS Code Server from my Proxmox host?

**Comment by tteck** [comment, marked as answer]
```
rm -rf ~/.local/lib/code-server- ~/.local/bin/code-server ~/.local/share/code-server ~/.config/code-server ~/.cache/code-server && reboot
```

**Reply by dimatx** [reply]
Thanks. I did run that, but VS Code Server is still loading on host port 8680. Do I need to do anything else?

**Reply by tteck** [reply]
reboot (added to line)
